tests/keras/test_callbacks.py

```diff
@@ -355,30 +352,22 @@ def data_generator_graph(train):
                                 batch_size=5)
     cbks = [tsb]
 
-    # fit with validation data
-    model.fit(X_train, y_train, batch_size=batch_size,
-              validation_data=(X_test, y_test), callbacks=cbks, epochs=3)
+    # fit without validation data
+    model.fit([X_train] * 2, [y_train] * 2, batch_size=batch_size,
+              callbacks=cbks, epochs=3)
 
     # fit with validation data and accuracy
-    model.fit(X_train, y_train, batch_size=batch_size,
-              validation_data=(X_test, y_test), callbacks=cbks, epochs=2)
-
-    # fit generator with validation data
-    model.fit_generator(data_generator(True), len(X_train), epochs=2,
-                        validation_data=(X_test, y_test),
-                        callbacks=cbks)
```

Comment:
That one was already redundant. It's same as fourth one below. Or you mean I shouldn't replace single input models with multi-input models and add a new test instead? I thought multi-input would already cover single-input cases.